### PR TITLE
add weapon add-on example

### PR DESCRIPTION
**Pull request by xxshady**

### CODE REVIEW

**Comment by Unnvaldr** [comment]
Hi,
could contribute that resource inside https://github.com/altmp/altv-example-resources repo?

**Review by Unnvaldr — APPROVED** [review]

**Comment by xxshady** [comment]
Ok, i will do it

**Comment by xxshady** [comment]
@Unnvaldr https://github.com/altmp/altv-example-resources/pull/14

File: articles/tutorials/stream_weapons.md [hunk 1]
```diff
@@ -10,7 +10,10 @@ This tutorial show, how to add/replace custom weapons for your server.
 
 ## <u>Weapons add-on</u>
 
+### Ready-to-use [example resource](https://github.com/altmp/altv-example-resources/tree/master/weapon-addon)
```
Comment (Unnvaldr) [inline articles/tutorials/stream_weapons.md:13]: Probably should be `Ready-to-use [example resource](https://github.com/altmp/altv-example-resources/tree/master/weapon-addon) based on infinity blade model.`?
Also don't consider it a header, cause it's not.

**Review by Unnvaldr — APPROVED** [review]

**Pull request merged**